app/Models/Contest.php

```diff
@@ -343,6 +347,11 @@ public function userEntries($user = null)
         );
     }
 
+    public function usersVotedCount(): int
+    {
+        return $this->votes()->distinct()->count('user_id');
```

Comment:
nonsense query aside as when read normally `distinct count(column)` is a noop (but still works somehow because laravel rewrites it; if anything it should be `distinct('user_id')->count()`), this should probably be cached like `entriesByType` otherwise it may take 40+ms on larger contests